server/app/routes/api/v1/va/serializers.js

```diff
@@ -99,8 +99,9 @@ function serializeAmbassadorForAdmin(ambassador) {
   obj["onboarding_completed"] = ambassador.get("onboarding_completed")
   obj["giftcard_completed"] = ambassador.get("giftcard_completed")
   obj["signup_completed"] = ambassador.get("signup_completed")
-  obj["locked"] = isLocked(ambassador);
+  obj["locked"] = isLocked(ambassador)
```

Comment:
I think you could avoid calling isLocked here since it's already set in serializeAmbassador

Reply:
removed